stock_move_location/wizard/stock_move_location_line.py

```diff
@@ -100,7 +102,7 @@ def _get_move_line_values(self, picking, move):
         self.ensure_one()
         location_dest_id = (
             self.move_location_wizard_id.apply_putaway_strategy
-            and self.destination_location_id.get_putaway_strategy(self.product_id).id
+            and self.destination_location_id._get_putaway_strategy(self.product_id).id
```

Comment:
From my first comment, this won't work as the destination location will always be the wizard one.